ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteBucket.java

```
@@ -64,13 +65,17 @@ public DeleteResult call() throws Exception {
         }
     }
 
-    private String deleteBucket() throws SignatureException, SSLSetupException, CommandException {
+    private String deleteBucket() throws SignatureException, SSLSetupException, CommandException, IOException {
         try {
             getClient().deleteBucket(new DeleteBucketRequest(bucketName));
         }
-        catch (final IOException e) {
+        catch (final FailedRequestException e) {
+            if (e.getStatusCode() == 409) { //BUCKET_NOT_EMPTY
+                return "Error: Tried to delete a non-empty bucket without the force delete objects flag.\nUse --force to delete all objects in the bucket";
```

This should return a command exception.